lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -583,6 +584,21 @@ module.exports = class MethodMappings {
           return value;
         },
 
+        async isElementPresent(webElementOrId) {
+          try {
+            const element = await this.getWebElement(webElementOrId);
```

Comment:
@piyushmishra1416 this method will be used with new element apis i.e. the argument will always be a web element and not the id. considering this can you please rename the argument name to `webElement` from `webElementOrId` and additionally, update the `const element = await this.getWebElement(webElementOrId)` to `const element = await webElement`

Please add a comment on the top of the function regarding this assumption

Please address the changes in the test cases accordingly. Comment back here once you are done with the changes and the testing

Reply:
fixed this

Reply:
> Please add a comment on the top of the function regarding this assumption
can you please address this as well

did you try running test cases again?

Reply:
Yes, and it was working fine for the elements that were present.